Update/ura_003.txt

```diff
@@ -547,7 +547,7 @@ void main()
 //r…まぁ、それでもこれを借りた時には、相当無理を言ってしまったのも確かだ。さすがにこれまでの貸し借りを差っ引いても、アイス程度では申し訳が立たないだろう。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…まぁ、それでもこれを借りた時には、相当無理を言ってしまったのも確かだ。さすがにこれまでの貸し借りを差っ引いても、アイス程度では申し訳が立たないだろう。",
-			NULL, "...Well, back when I told her I was borrowing it, I'm sure she did say it was unreasonable. Even if you subtract the things I offered her in exchange, I really left her on the ice. I felt sorry about that.", Line_Normal);
+			NULL, "...Well, back when I told her I was borrowing it, I'm sure she did say it was unreasonable. I'm sure that even if you subtracted all the loans and debt I'd accumulated, I wouldn't be able to make it up to her with merely ice cream.", Line_Normal);
```

Comment:
```suggestion
			NULL, "...Well, back when I told her I was borrowing it, I'm sure she did say it was unreasonable. I'm sure that even if you subtracted all the loans and debt I'd accumulated, it would take more than just ice cream to pay her back.", Line_Normal);
```